docs/content/releases.md

```diff
@@ -6,6 +6,72 @@ doctypes: ["concept"]
 toc: true
 docs: "DOCS-616"
 ---
+
+## NGINX Ingress Controller 3.0.0
+
+12 January 2023
+
+OVERVIEW:
+
+* *Support for discovery.k8s.io/v1/EndpointSlices*. NGINX Ingress Controller 3.0.0 removes support for `k8s.io/v1/Endpoints` API in favor of `discovery.k8s.io/v1/EndpointSlices`. This now means that the minimum supported version of Kubernetes is now 1.21. For older Kubernetes versions, use the 2.4.x release of the Ingress Controller. For details see [add support for EndpointSlices](https://github.com/nginxinc/kubernetes-ingress/pull/3260).
```

Comment:
the title should be "the minimum supported version of Kubernetes is now 1.21" not *Support for discovery.k8s.io/v1/EndpointSlices*